lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -24,6 +24,14 @@ defmodule Mix.Tasks.Xref do
     * `--no-archives-check` - do not check archives
     * `--no-elixir-version-check` - do not check the Elixir version from mix.exs
 
+  ## Configuration
+
+    All configuration for Xref should be placed under the key `:xref`.
+
+    * `:exclude` - modules and `{module, func, arity}`s to ignore when checking
+      cross references. For example:
+      `xref: [exclude: [MissingModule, {MissingModule2, :missing_func, 2}]]`
```

Comment:
I would not show the whole `:xref -> :exclude -> ...` example as we're describing just the `:exclude` key. I'd just show "For example `[MissingModule, {OtherMissingModule, :missing_func, 2}]`".